### PR TITLE
Changes HTTP verbs enum to lowercase

### PR DESCRIPTION
**Pull request by pedrovereza**
As discussed in #684

```swift
public enum Method: String {

    case options = "OPTIONS"
    case get     = "GET"
    case head    = "HEAD"
    case post    = "POST"
    case put     = "PUT"
    case patch   = "PATCH"
    case delete  = "DELETE"
    case trace   = "TRACE"
    case connect = "CONNECT"
```

### CODE REVIEW

**Comment by BasThomas** [comment]
Was just busy doing this, thank god I had a GitHub window open 🙈

**Comment by pedrovereza** [comment]
@BasThomas Oh, my bad! Totally forgot to mention I was working on it 😅

File: Source/Target.swift [hunk 1]
```diff
@@ -49,14 +49,23 @@ public enum StructTarget: TargetType {
 
 /// Represents an HTTP method.
 public enum Method: String {
-    case GET, POST, PUT, DELETE, OPTIONS, HEAD, PATCH, TRACE, CONNECT
+
+    case options = "OPTIONS"
```
Comment (BasThomas) [inline Source/Target.swift]: Any reason for swapping this order here? Not that it matters that much, just curious

**Comment by BasThomas** [comment]
No worries!

File: Source/Target.swift (hunk 1 above)
Comment (pedrovereza) [inline Source/Target.swift]: No reason, was using [Alamofire as a reference](https://github.com/Alamofire/Alamofire/blob/e8bca7e40a861327366c5ccba42a688d17ec482a/Source/ParameterEncoding.swift#L30-L40)

Should I reorder it back to how it was?

Comment (sunshinejr) [inline Source/Target.swift]: Hmm, didn't really check the code so forgive me, but is there a reason we cannot use typealias for existing enum from Alamofire?

Comment (pedrovereza) [inline Source/Target.swift]: Tried this and seems to work fine.

```swift
public typealias Method = Alamofire.HTTPMethod

extension Method {
    public var supportsMultipart: Bool {
        switch self {
        case .post,
             .put,
             .patch,
             .connect:
            return true
        default:
            return false
        }
    }
}
``` 

Only downside I can see here is that it's yet another dependency on Alamofire. But given the discussion in #556, not sure if that's an actual issue.

Comment (BasThomas) [inline Source/Target.swift]: Yeah, we might want to just use a typealias here.

Comment (pedrovereza) [inline Source/Target.swift]: Done 😉

**Pull request merged**